### PR TITLE
gh-105063: Disable test_peg_generator.TestCParser bco. ref leaks

### PR DESCRIPTION
**Pull request by erlend-aasland**
Since gh-104798 (Use setuptools in peg-generator and reenable
tests), the TestCParser test case has been producing ref leaks.

* Issue: gh-105063

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Let's wait for gh-106013 to be merged, before landing this.

**Review by lysnikolaou — APPROVED** [review]

**Pull request merged**
